lib/RemoteControl/src/App.cpp

```diff
@@ -160,27 +160,41 @@ void App::setup()
         String   mqttAddr = Settings::getInstance().getMqttBrokerAddress();
         uint16_t mqttPort = Settings::getInstance().getMqttPort();
 
-        /* Setup MQTT Server, Birth and Will messages. */
-        if (false ==
-            Board::getInstance().getNetwork().setConfig(clientId, ssid, password, mqttAddr, mqttPort, TOPIC_NAME_BIRTH,
-                                                        String(clientId + String(" Connected!")), TOPIC_NAME_WILL,
-                                                        String(clientId + String(" Disconnected!")), true))
+        /* Setup Network. */
+        NetworkSettings settings = {ssid, password, String("Zumo_AP"), String("")};
+        if (false == Board::getInstance().getNetwork().setConfig(settings))
         {
             LOG_ERROR("Network Configuration could not be set.");
             fatalErrorHandler();
         }
 
+        /* Setup MQTT Server, Birth and Will messages. */
+        m_mqttClient.init();
+        MqttSettings mqttSettings = {clientId,
+                                     mqttAddr,
+                                     mqttPort,
+                                     TOPIC_NAME_BIRTH,
+                                     String(clientId + String(" Connected!")),

```

Comment:
Would it be possible to change the birth and will messages to JSON format? Something like:
``` json
{ "name" : clientId}
```
This could be useful for later applications.
The same message can be used as birth and will.